### DISCUSSION TITLE
video source 0 ok, video source 2 not

### DISCUSSION THREAD

**Discussion by flipper2222222** [GerryDazoo/Slinger#409] [General]
hi everyone, 

please see my log below. extreme player on IOS iPhone is fine for both sources, but VLC on windows drops out just after connecting

composite source 0 is fine with vlc on windows but source 2 which is hdmi with an old TiVo connected drops out, 

anyone have any idea why or which setting I might adjust.

also besides KM player any other windows alternatives to VLC? thanks in advance 

Slinginfo  "350/500/M1" 12 60 192.168.0.3 5216 3072 3
500t Starting Stream for  192.168.0.1:22187
Connecting... ('192.168.0.3', 5216) Control
500t Selecting Video Source 2
Keycodes= [4, 5, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 24, 26, 27, 28, 29, 32, 35, 38, 39, 40, 41, 42, 46, 50, 62, 63, 64, 65, 66, 67, 68, 69, 70, 74, 75, 76, 77]
VideoParameters: Resolution= 12 FrameRate= 60 VideoBandwidth= 2000 VideoSmoothness= 63 IframeRate= 5 AudioBitRate= 64
Connecting... ('192.168.0.3', 5216) Stream
500t Stream started at 12/04/2022, 19:26:48.567  749 0
12/04/2022, 19:26:55.143  500t Stream Terminated for  192.168.0.1:22187 [WinError 10053] An established connection was aborted by the software in your host machine
500t Shutting down connections
500t Logging Out

**Comment by flipper2222222** [comment]
smplayer fixed this problems no more drop out on my 500 hdmi

**Reply by sp222222** [reply]
you're welcome :) - close this one GD

**Comment by flipper2222222** [comment]
this was answered on Discord by user   Lefty_Knavey#9033 thank you
